io.itch.drwhut.TabletopClub.json

```
@@ -0,0 +1,60 @@
+{
+    "app-id": "io.itch.drwhut.TabletopClub",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tabletop-club-wrapper",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
```

```suggestion
        "--socket=x11",
```

It doesn't work on Wayland.

---

Thank you for the heads up - I had thought initially that adding `--socket=wayland` would have worked since the standalone version runs fine under Wayland, but that is only because it's using XWayland.

I have pushed a new commit with this change.